plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/deserializer/InboundEndpointDeserializer.java

```diff
@@ -721,7 +729,28 @@ private void addParameterForInboundEndpoint(Map.Entry<String, String> paramEntry
                         } else {
                             executeSetValueCommand(INBOUND_ENDPOINT__CONTENT_TYPE, ContentType.JSON);
                         }
-                    }
+                    } else if (paramEntry.getKey().equals(InboundEndpointConstants.INBOUND_MQTT_SSL_KEYSTORE_LOCATION)) {
+	                 	executeSetValueCommand(INBOUND_ENDPOINT__TRANSPORT_MQTT_SSL_KEYSTORE_LOCATION, 
+	                 			paramEntry.getValue());                    	
+	                } else if (paramEntry.getKey().equals(InboundEndpointConstants.INBOUND_MQTT_SSL_KEYSTORE_TYPE)) {
```

Comment:
Wouldn't it be clearer if we have a switch instead?

Reply:
Yes it definitely would. But the existing code base is having only if-else. Therefore, just followed the convention.

Reply:
Shall we fix the formatting issues here?